### PR TITLE
fix some issues

### PR DESCRIPTION
**Pull request by mmattel**
This PR fixes some general page issues

### CODE REVIEW

File: src/partials/toolbar.hbs [hunk 1]
```diff
@@ -6,6 +6,6 @@
 {{> breadcrumbs}}
 {{> page-versions}}
   {{#if page.editUrl}}
-  <div class="edit-this-page"><a href="{{page.editUrl}}">Edit this Page</a></div>
+  {{!-- <div class="edit-this-page"><a href="{{page.editUrl}}">Edit this Page</a></div> --}}
```
Comment (settermjd) [inline src/partials/toolbar.hbs]: If we're removing this, why not remove it completely? Or did I forget something that we discussed, earlier today?

Comment (tboerger) [reply src/partials/toolbar.hbs]: Why do you want to remove it at all?

**Comment by mmattel** [comment]
Commented out instead of removing to keep it as an example

**Comment by mmattel** [comment]
@tboerger I have highlighted explicitly in the footer under topic "Interact" --> "Contribute" (which points clearly and directly to the various contribution possibilities including documentation) and removed on "EVERY" page "Edit this Page".

**Comment by tboerger** [comment]
> @tboerger I have highlighted explicitly in the footer under topic "Interact" --> "Contribute" (which points clearly and directly to the various contribution possibilities including documentation) and removed on "EVERY" page "Edit this Page".

This is still something I disagree on. The `Edit this page` link is for me pretty useful and that should be decided by multiple people, not only by you or @settermjd ;)

**Comment by mmattel** [comment]
Whom do you want to add ?

File: src/partials/toolbar.hbs (hunk 1 above)
Comment (settermjd) [reply src/partials/toolbar.hbs]: @mmattel, can you add your reasoning here, pls?

**Comment by mmattel** [comment]
@tboerger @settermjd 
Summarizing the reason for removing ``Edit this Page`` on ``every page`` and renaming in the footer ``Interact: Get Involved`` to ``Interact: Contribute`` is quite simple. **Preventing being recognized as tinker kiosk.** After being decades in enterprise environments I can proof that the documentation is a major part how a company presents itself – it is a businesscard. You will find on the majority of homepages the reference to feedback – however it is made – but not on every page the reader gets presented. How does ownCloud wants to be percieved, professional or a tinker kiosk? It is for sure that ``Edit this Page`` on ``every page`` will generate a different perception than a good visible note on the footer to contribute – which btw directly points to a page where the various contribution possibilities are described including documentation. 

First time readers get a clear direction where to contribute and recurring contributers do not need it anyways.

File: src/partials/toolbar.hbs (hunk 1 above)
Comment (mmattel) [reply src/partials/toolbar.hbs]: pls see below

**Comment by tboerger** [comment]
Sorry, but calling a company a tinker kiosk because it provides a link to directly edit the page is bullshit for me. It absolutely reduces the effort how to fix typos or to add suggestions/improvements.

**Comment by mmattel** [comment]
@tboerger the discussion it is about how readers can percieve this kind of link, nothing else.

@felixboehm @hodyroff 
Sorry involving you. After having a talk with @settermjd I would like to ask you for your advice.
The discussion here is about having "Edit this Page" on every docs page or remove that (currently dead) link and have a more descriptive single footer link instead or any combination, please see the screenshots below:

Many thanks in advance

Keeping and fixing / removing Edit this Page:
image

Adding Contribute in the footer (links to https://owncloud.org/community/)
image

**Comment by mmattel** [comment]
@felixboehm @hodyroff,
mind to add your comment so we can finalize

**Comment by felixboehm** [comment]
The link to edit the current page activates to contribute easily, why not? I like this personally very much, and see no disadvantage.
The link should work of course.

**Comment by settermjd** [comment]
Thanks for the feedback, @felixboehm.

**Pull request closed without merging**